### PR TITLE
[WebGL] upgrade tfjs-core version to 1.2.0

### PR DESCRIPTION
**Pull request by GreyZzzzzzXh**

### CODE REVIEW

File: src/nn/webgl/WebGLModel.js [hunk 1]
```diff
@@ -689,6 +689,7 @@ export default class WebGLModel {
   }
 
   static _supportWebGL() {
+    tf.backend();
```
Comment (huningxin) [inline src/nn/webgl/WebGLModel.js]: what does this call do?

Comment (GreyZzzzzzXh) [reply src/nn/webgl/WebGLModel.js]: In the latest version, `tf.getBackend()`  returns `undefined` if we do nothing before it. 

`tf.backend()` will register webgl backend if no backend has been registered.
Changed this to `tf.setBackend('webgl');` to be clearer.

**Comment by GreyZzzzzzXh** [comment]
Found a memory leak problem..  Reopen this PR after fix it.

**Comment by GreyZzzzzzXh** [comment]
> Found a memory leak problem..

The memory leak problem is not caused by this PR, I will open an issue and solve it.

**Comment by huningxin** [comment]
Looks good. Thanks.

**Pull request merged**